app/api/src/validations/taskValidation.ts

```diff
@@ -10,9 +10,6 @@ export const createTaskValidation = (data: TaskInterface) => {
       .valid('Top Priority', 'Important', 'Other')
       .required(),
     timers: Joi.number().min(1).required(),
-    date: Joi.date().required(),
-    user_id: Joi.string().required(),
```

Comment:
wait james, why are we removing date, user_id, and _id from the create Task validation -> this is going to throw off abdul's front end create task service